kubectl-cloudflow/deploy/deploy_test.go

```diff
@@ -1,103 +1,273 @@
 package deploy
 
 import (
-	"encoding/json"
 	"fmt"
-	"os"
+	"encoding/json"
 	"strings"
-	"testing"
 
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/go-akka/configuration"
 	"github.com/lightbend/cloudflow/kubectl-cloudflow/domain"
-	"github.com/stretchr/testify/assert"
 )
 
-func TestMain(m *testing.M) {
-	code := m.Run()
-	os.Exit(code)
+func createBlueprint() string {
+	return `blueprint {
```

Comment:
very minor, start with newline so blueprint is a little nicer to read?

Reply:
done